src/screens/users/list/Users.tsx

```diff
@@ -46,9 +49,20 @@ export default class Users extends BaseAutoRefreshScreen<Props, State> {
     this.setRefreshPeriodMillis(Constants.AUTO_REFRESH_LONG_PERIOD_MILLIS);
   }
 
+  public async componentDidMount(): Promise<void> {
+    this.userIDs = Utils.getParamFromNavigation(this.props.route, 'userIDs', null) as string;
+    this.carFullName = Utils.getParamFromNavigation(this.props.route, 'carFullName', null) as string;
+    this.carLicensePlate = Utils.getParamFromNavigation(this.props.route, 'carLicensePlate', null) as string;
+    await super.componentDidMount();
+  }
+
   public async getUsers(searchText: string, skip: number, limit: number): Promise<DataResult<User>> {
     try {
-      const users = await this.centralServerProvider.getUsers({ Search: searchText }, { skip, limit });
+      const params = {
+        Search: searchText,
+        UserID: this.userIDs
```

Comment:
The join with the '|' should be done here.

Reply:
Do you want the join with '|' to be at the UserID level ? If so, it is already done in the CarComponent line 44